UPGRADE-2.8.md

```diff
@@ -511,5 +511,5 @@ Yaml
 
  * Deprecated usage of a colon in an unquoted mapping value
  * Deprecated usage of `@`, `` ` ``, `|`, and `>` at the beginning of an unquoted string
- * Deprecated non-escaped \ in double-quoted strings when parsing Yaml
-   ("Foo\Var" is not valid whereas "Foo\\Var" is)
+ * Deprecated non-escaped `\` in double-quoted strings when parsing Yaml
+   (`Foo\Var` is not valid whereas `Foo\\Var` is)
```

Comment:
It's not super clear to me. Maybe a before and after? I believe it would be:

Before:
```yml
class: "Foo\Var"
```

After
```yml
class: "Foo\\Var"
```

Or, you could use single quotes and just have `Foo\Var`, right? Can you have no quotes and just `Foo\Var`?